doc/guides/abstraction/posix.rst

```diff
@@ -0,0 +1,434 @@
+
+
+POSIX Support
+#############
+
+The Portable Operating System Interface (POSIX) is a family of standards
+specified by the IEEE Computer Society for maintaining compatibility between
+operating systems. Zephyr implements a subset of the embedded profiles PSE51
+and PSE52, and BSD Sockets API.
+
+With the POSIX support available in Zephyr, an existing POSIX compliant
+application can be ported to run on the Zephyr kernel, and therefore leverage
+Zephyr features and functionality. Additionally, a library designed for use with
+POSIX threading compatible operating systems can be ported to Zephyr kernel
+based applications with minimal or no changes.
+
+..  figure:: posix.svg
+    :align: center
+    :alt: POSIX Support in Zephyr
+
+    POSIX support in Zephyr
+
+The POSIX API subset is an increasingly popular OSAL (operating system
+abstraction layer) for IoT and embedded applications, as can be seen in
+Zephyr, AWS:FreeRTOS, TI-RTOS, and NuttX.
+
+Benefits of POSIX support in Zephyr include:
+
+- Offering a familiar API to non-embedded programmers, especially from Linux
+- Enabling reuse (portability) of existing libraries based on POSIX APIs
+- Providing an efficient API subset appropriate for small (MCU) embedded systems
+
+
+System Overview
+===============
+
+Units of Functionality
+++++++++++++++++++++++
+
+The system profile is defined in terms of component profiles that specify Units
+of Functionality that can be combined to realize the application platform. A Unit
+of Functionality is a defined set of services which can be implemented. If
+implemented, the standard prescribes that all services in the Unit must
+be implemented.
+
+A Minimal Realtime System Profile implementation must support the
+following Units of Functionality as defined in IEEE Std. 1003.1 (also referred to
+as POSIX.1).
+
+
+.. csv-table:: Units of Functionality
+   :header: Requirements, Supported, Remarks
+   :widths: 50,10,60
+
+
+    POSIX_C_LANG_JUMP,|crossmark|
```

Comment:
The checkmark ✓ and crossmark ❌ unicode characters chosen are unfortunately not similarly sized (I can't find a nicely sized green checkmark).  I've also had issues with Asian readers where X typically means "yes" and O means "no".  In these tables, it would be better to just say "Yes" and "No" instead of using unicode symbols.

Reply:
removed unicode